traffic_ops/traffic_ops_golang/routing/middleware/wrappers.go

```diff
@@ -39,23 +39,39 @@ import (
 	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/tocookie"
 )
 
+const DefaultRequestTimeout = time.Second * time.Duration(60)
+
+// Middleware ...
```

Comment:
While you're moving things around, do you mind adding godoc comments to `Middleware`, `GetDefault`, and `Use`?

Reply:
Done. The middleware package now has full GoDoc.

I also moved the RFC and utility symbols that were in there to lib/go-util and lib/go-rfc where they belong. Some of the RFC funcs were using RFC things in lib/go-tc, so I also had to move those to lib/go-rfc. It was a string I didn't intend to pull, but it needed done anyway, and it's done now.

If it's easier to review, I made a new changeset, you should be able to just review https://github.com/apache/trafficcontrol/pull/4171/commits/44bce47c04df52f3a7721f62ee34c8ffd0f19998 - I didn't modify the original changeset.